package.json

```diff
@@ -22,24 +22,25 @@
   },
   "main": "./lib/index.js",
   "scripts": {
-    "test": "./node_modules/.bin/mocha test/",
-    "test-browser": "./node_modules/.bin/mocha test/browser/"
+    "mocha": "mocha",
```

Comment:
what is this script for?

Reply:
Otherwise, the script points to "./node_modules/.bin/mocha", which is bad. It let's npm or yarn work out which mocha binary to use.